### ISSUE TITLE
In project creation, you can select an owner who doesn't have authorization to the Rancher instance and will not be able to access the project

### ISSUE THREAD

**Issue by deniseschannon** [rancherio/rancher#189]
Currently, whenever we create a project, there is the ability to select any team associated with the logged in GitHub account. There are team members in that team list that may not have access to the Rancher instance and will not be able to access Rancher and the project. 

There are a couple options of what could be done to fix it. 

Option 1: When creating projects, any owner (team or user) that has been added to a project space will automatically be authorized as a user/organization for the Rancher instance.

Option 2: When creating projects, the list of "Choose an Owner" should only provide the list of users/organizations that has been authorized in the Access Control.

**Comment by ibuildthecloud** [comment]
I vote for option 1, 2 is confusing to the user.

**Comment by vincent99** [comment]
We're adding support for multiple users/teams/orgs.  Once that's there it will be more like the access control page where the list of allowed users will default to you but can be added to.

**Comment by smw355** [comment]
+1

Best Regards,

Shannon Williams
shannon@rancher.com<mailto:shannon@rancher.com>
+1 650-521-6902
www.rancher.com<http://t.signaledue.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX48p_23-W1px3nY2BpKfMW64zBkx56dz44f5D73Wx02?t=http%3a%2f%2fwww.rancher.com&si=4781742672576512&pi=7370e19c-2820-4cd2-8ae3-dffea6d0d48b>

**Comment by hibooboo2** [comment]
I will work on this as part of #194

**Comment by hibooboo2** [comment]
After a design meeting this is what it seems like we should go with. 

# Access
1. Unrestricted Auth. 
    * Anyone with a github account can access the rancher instance and use the site as a user.
*  Restricted Auth
    * Anyone with Github account can view Rancher instance, but no ability to do anything.
    * They are provided with a **Must contact Admin** to request being enabled as a user.
    * When enabled the can use rancher as a user.

# Projects
* Any user can create a project.
* Anyone that creates a project is automatically the owner to the project they created.

## Project User Roles
1. Owner
    * Invite members and delegate owners.
2. Member
    * CRUD on containers / hosts
    * Cannot delegate owners or add members.

## Inviting users (Owners/ Members to a project
Owner defines if a user is a owner or member of the project.
### Access control option 1
The user can access rancher instance and access the project.
### Access control option 2
* If user being invited is already enabled they now have access to the project as well as the rancher instance.
* If the user being invited is not already enabled for the rancher instance they are granted access to the project and not the Rancher instance.

**Comment by hibooboo2** [comment]
@wlan0 @vincent99 @sangeethah @will-chan @deniseschannon Any feed back or are we in agreement on this?

**Comment by vincent99** [comment]
TL;DR: Users who are added as a member of a project are automatically granted access to Rancher now.

**Comment by vincent99** [comment]
**Restricted Auth**: If auth is restricted and the user is not authorized (by the whitelist or being a member of a project) then they can see the login page and will just get "Unauthorized" when clicking it.

**Comment by tfiduccia** [comment]
v0.18.1
Verified fixed
